src/blocks/heroes/FullScreenHeroWithPromoImagesAndTypedText/FullScreenHeroWithPromoImagesAndTypedText.js

```diff
@@ -221,7 +222,7 @@ const FullScreenHeroWithPromoImagesAndTypedText = ({
               position={'absolute'}
               sx={{ transform: 'translate3d(20%, -50%, 0)' }}
             >
-              {imageList.map((item, i) => (
+              {mockimages.map((item, i) => (
```

Comment:
Will need hello and meeting on how to move forward for this one, using mock data for now to fix broken Solutions page